charts/hub-agent/templates/deployment.yaml

```diff
@@ -27,12 +27,8 @@ spec:
             - --webhook-client-connection-type={{.Values.webhookClientConnectionType}}
             - --v={{ .Values.logVerbosity }}
             - -add_dir_header
-            {{- if .Values.enableV1Alpha1APIs }}
-            - --enable-v1alpha1-apis
-            {{- end }}
-            {{- if .Values.enableV1Beta1APIs }}
```

Comment:
hi Chen, just wonder why the original one is not correct, thanks

Reply:
Hi Zhiying! The args have default values (`true` for v1alpha1 and `false` for v1beta1); in the old setup there is no way to disable v1alpha1 and enable only v1beta1, even if we set `enableV1Alpha1APIs` to `false`.